src/monogame/vulkan/RebuildMGFX.bat

```diff
@@ -0,0 +1,19 @@
+@echo off
```

Comment:
Decided it was best to keep the generated binary effect files in per-platform folders instead of all in a common place.

This seems like a good direction considering console support.